cordova-lib/src/cordova/plugin.js

```diff
@@ -206,7 +207,7 @@ module.exports = function plugin(command, targets, opts) {
                         });
                     }, Q())
                     .then(function() {
-                        return plugman.raw.uninstall.uninstallPlugin(target, path.join(projectRoot, 'plugins'));
+                        return plugman.raw.uninstall.uninstallPlugin(target, path.join(projectRoot, 'plugins')).then(savePluginsToConfig());
```

Comment:
I think here and below you actually want: `.then(savePluginsToConfig)` rather than `.then(savePluginsToConfig())`. Often with promises, it's best to just always use `.then(function() {})`, even if all it does is delegate to a helper.